### PR TITLE
Fixed comment of getSourceLocation()'s returns

### PR DESCRIPTION
**Pull request by jiakuan**

### CODE REVIEW

**Comment by ggrossetie** [comment]
Thanks!
Could you please also fix the same mistake on the type definition?

https://github.com/asciidoctor/asciidoctor.js/blob/a8c4c96829975c7f47c9fab59d5f35bddd7535da/packages/core/types/index.d.ts#L2082

**Comment by jiakuan** [comment]
Done

File: packages/core/src/asciidoctor-core-api.js [hunk 1]
```diff
@@ -587,7 +587,7 @@ AbstractBlock.prototype.setStyle = function (style) {
 /**
  * Get the location in the AsciiDoc source where this block begins.
  *
- * @returns {string} - the style for this block
+ * @returns {string} - the location in the AsciiDoc source where this block begins
```
Comment (ggrossetie) [inline packages/core/src/asciidoctor-core-api.js:590]:
```suggestion
 * @returns {Cursor} - the location in the AsciiDoc source where this block begins
```

**Comment by ggrossetie** [comment]
Awesome, thanks ❤️

**Pull request merged**